lib/ex_cycle/validations/lock.ex

```diff
@@ -76,7 +76,7 @@ defmodule ExCycle.Validations.Lock do
     end)
   end
 
-  def next(state, %Lock{unit: :week_day}) do
+  def next(%ExCycle.State{} = state, %Lock{unit: :week_day}) do
```

Comment:
did this change sneak in accidentally?

Reply:
No, `dialyzer` complained about the `else` condition of this function without this change.